docs/development/building-and-testing-packages.md

```diff
@@ -1,27 +1,65 @@
-(building-and-testing-packages)=
+(building-and-testing-packages-out-of-tree)=
 
 # Building and testing Python packages out of tree
 
 This is some information about how to build and test Python packages against
 Pyodide out of tree (for instance in your package's CI or for use with private
 packages).
 
+Pyodide currently only supports Linux for out of tree builds, though there is a
+good change it will work in MacOS too. If you are using Windows, try Windows
+Subsystem for Linux.
+
 ## Building binary packages for Pyodide
 
 If your package is a pure Python package (i.e., if the wheel ends in
 `py3-none-any.whl`) then follow the official PyPA documentation on building
 [wheels](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives)
-Otherwise, the procedure is simple. In your package directory run the following
-command line commands:
+Otherwise, the procedure is simple.
```

Comment:
Maybe we shouldn't say it is "simple" since in some circumstances it will not be simple (and even when it is, not everyone will find it to be),

Reply:
Right, we should not judge readers. Thanks!